aap-common/proc-aap-list-available-playbooks.adoc

```diff
@@ -48,7 +48,7 @@ Description: Add extension nodes to an existing Ansible Automation Platform from
 -----------------------------------------------
 This playbook is used to deploy extension nodes to an existing Ansible Automation Platform from AWS Marketplace environment.
 For more information regarding extension nodes, visit our official documentation -
-https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/red_hat_ansible_automation_platform_from_aws_marketplace_guide/assembly-aap-aws-deploy-extension
+https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/red_hat_ansible_automation_platform_from_aws_marketplace_guide/assembly-aap-aws-extension
```

Comment:
This ought to be:
For more information on extension nodes, see link:https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/red_hat_ansible_automation_platform_from_aws_marketplace_guide/assembly-aap-aws-extension[WHATEVER THE TITLE IS]

Reply:
This isn't a proper link in the main body of the docs: it's just an output of a docker command, so I don't think it needs the [title]